MapsSDKExamplesKotlin/app/src/main/java/com/trimblemaps/mapssdkexampleskotlin/SampleBasicMapActivity.kt

```diff
@@ -0,0 +1,44 @@
+package com.trimblemaps.mapssdkexampleskotlin
+
+import android.app.Activity
+import android.os.Bundle
+import com.trimblemaps.account.LicensedFeature
+import com.trimblemaps.account.TrimbleMapsAccountManager
+import com.trimblemaps.account.models.TrimbleMapsAccount
+import com.trimblemaps.mapsdk.TrimbleMaps
+import com.trimblemaps.mapsdk.maps.MapView
+import com.trimblemaps.mapsdk.maps.OnMapReadyCallback
+import com.trimblemaps.mapsdk.maps.Style
+
+
+class SampleBasicMapActivity : Activity() {
+    private var mapView: MapView? = null
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        // Authorize the api key for the session.
+        // .apiKey() requires your Trimble Maps API key
+        val trimbleMapsAccount = TrimbleMapsAccount.builder()
```

Comment:
Instead of all the independent activities doing account initialization. It'd make sense to just have this done once on startup. Likely the main activity.

Reply:
I did it intentionally, to keep the code of each example as similar as possible to the code shown in the documentation.
But I understand that it looks better with just one account initialization
I'll update it.